Mathlib/Data/NNRat/Defs.lean

```diff
@@ -212,6 +223,10 @@ theorem sub_def (p q : ℚ≥0) : p - q = toNNRat (p - q) :=
 theorem abs_coe (q : ℚ≥0) : |(q : ℚ)| = q :=
   abs_of_nonneg q.2
 
+@[simp 1100]
```

Comment:
Can you add a note explaining the increased priority?

Reply:
Will do. I'll be back at a computer in 15-30 minutes

Reply:
Yael added a library note for this. I referenced that and just made the priority `high` for simplicity.

Reply:
Done